### ISSUE TITLE
Compatibility issues with JS API 4.0

### ISSUE THREAD

**Issue by patrickarlt** [Esri/calcite-web#580]
Drop the following in the [JS API sandbox](https://developers.arcgis.com/javascript/latest/sample-code/sandbox/sandbox.html?sample=get-started-mapview) and hit "Run".

```html
<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no">
  <title>Get started with MapView - Create a 2D map - 4.0</title>
  <style>
    html,
    body,
    #viewDiv {
      padding: 0;
      margin: 0;
      height: 100%;
      width: 100%;
    }
  </style>
  <link rel="stylesheet" href="https://js.arcgis.com/4.0/esri/css/main.css">
  <link rel="stylesheet" href="https://s3-us-west-1.amazonaws.com/patterns.esri.com/files/calcite-web/1.0.0-beta.14/css/calcite-web.css">
  <script src="https://js.arcgis.com/4.0/"></script>

  <script>
    require([
      "esri/Map",
      "esri/views/MapView",
      "dojo/domReady!"
    ], function(Map, MapView) {

      var map = new Map({
        basemap: "streets"
      });

      var view = new MapView({
        container: "viewDiv",
        map: map,
        zoom: 4,
        center: [15, 65]
      });

    });
  </script>
</head>

<body>
  <div id="viewDiv"></div>
</body>
</html>
```

* [ ] There are no map tiles
* [ ] Zoom buttons do not appear in the correct places

How flip the order of the JS API CSS and Calcite Web. Hit "Run" again:

* [ ] Missing characters in the zoom boxes

**Comment by paulcpederson** [comment]
the second two are due to declaring their classes in the icon font section, which we can just remove if they are supplying their own icon definition/font.

The first issue should be resolved in their repo by adding `max-width: none;` on those images

**Comment by nikolaswise** [comment]
Agree with Paul. JS API should add that max width on the selector `.esri-view .esri-bitmap`

Also agree that we can pull the `.esri-icon` selector, since these icon sets have Officially Diverged

**Comment by nikolaswise** [comment]
Collisions removed in #581

**Comment by bsvensson** [comment]
@nikolaswise @paulcpederson - Sorry, I'm not following. Could you expand on what you think should be changed in JSAPI4?

**Comment by paulcpederson** [comment]
`.esri-bitmap` should have `max-width: none`

**Comment by paulcpederson** [comment]
essentially it's a very common strategy to add `max-width: 100%` on images (http://www.w3schools.com/css/css_rwd_images.asp) Because this is usually declared on the `img` tag, it will cause all the tiles to break.

**Comment by ycabon** [comment]
SGTM

**Comment by ycabon** [comment]
https://devtopia.esri.com/WebGIS/arcgis-js-api/issues/4888

**Comment by bsvensson** [comment]
FYI, this has been fixed for JSAPI 4.1 (planned for September release):
```
   .esri-bitmap {
      max-width: none;
      transform-origin: top left;
    }
```
